tests/contract_testcases/solana/selectors-must-be-different.sol

```diff
@@ -1,32 +1,40 @@
 abstract contract c {
-	function f1() selector=hex"01" public {}
+	@selector([1])
+	function f1() public {}
 	// error: selector must be unique
-	function f2() selector=hex"01" public {}
+	@selector([1])
+	function f2() public {}
 	// error: selector must be unique
-	function f3() selector=hex"01" public {}
+	@selector([1])
+	function f3() public {}
 }
 
 contract d {
 	int public c;
 
 	// error: selector is the same as c
-	function f1() selector=hex"c3da42b8" public {}
+	@selector([0xc3,0xda,0x42,0xb8])
```

Comment:
Please, add a space after the comma (for all the cases in this example)

Reply:
Thanks for fixing all these - but this is test data, not terribly important. On the other hand, examples should be nicely formatted of course.